xio-core/src/main/java/com/xjeffrose/xio/http/ProxyHandler.java

```diff
@@ -5,11 +5,15 @@
 import java.util.Optional;
 import lombok.extern.slf4j.Slf4j;
 
+import static com.xjeffrose.xio.http.ProxyRouteConfig.ProxyHostPolicy.UseConfigValue;
+import static com.xjeffrose.xio.http.ProxyRouteConfig.ProxyHostPolicy.UseRemoteAddress;
+import static com.xjeffrose.xio.http.ProxyRouteConfig.ProxyHostPolicy.UseRequestHeader;
+
```

Comment:
I'm assuming this is something IntelliJ did? we can get rid of these imports.

Reply:
Good call. Done.